packages/@ama-sdk/core/src/utils/encoder.ts

```diff
@@ -10,7 +10,7 @@ class Stream {
   private readonly tokens: number[];
 
   constructor(tokens: Uint8Array | number[]) {
-    this.tokens = [].slice.call(tokens);
+    this.tokens = Array.prototype.slice.call(tokens);
```

Comment:
I find it more clear with the empty array

Reply:
rule `unicorn/prefer-prototype-methods`